### ISSUE TITLE
Semantics for the reference used alone

### ISSUE THREAD

**Issue by chaosprint** [chaosprint/glicol#50]
In Glicol current implementation, the reference can be used alone.

```
~trigger: imp 1.0;
out: ~trigger >> sp \808bd_0
```

This is more like a syntax suger for:
```
~trigger: imp 1.0;
out: const_sig ~trigger >> sp \808bd_0
```
However, this does not work:
```
~tri: envperc 0.001 0.1

~lead: saw 100.0 >> mul ~tri

out: imp 1.0 >> ~tri
```
It seems the `ref` when used alone, can only output signal, but cannot take any input.
In the case that does not work above, should we connect the `imp 0.1` to the beginning of `~tri` chain?
But this is not good.
As it is still not making any sound.

**Comment by chaosprint** [comment]
Understand it in this way and imagine another syntax:
```
imp 1.0 >> send [~trigger]
get ~trigger >> sp \808bd_0 >> send [out]
```

**Comment by chaosprint** [comment]
fixed with #67 

`~some` is just a syntax sugar for `get ~some`
